packages/node_modules/@ciscospark/redux-module-presence/src/actions.js

```diff
@@ -155,22 +105,18 @@ export function subscribeToUpdates(userIds, spark) {
  * @param {object} spark
  * @returns {Thunk}
  */
-export function unsubscribeFromUpdates(userIds, spark) {
+export function unsubscribeFromPresenceUpdates(userIds, spark) {
   const subjects = Array.isArray(userIds) ? userIds : [userIds];
   return (dispatch) =>
-    spark.request({
-      method: `POST`,
-      api: `apheleia`,
-      resource: `subscriptions`,
-      body: {
-        subjects,
-        subscriptionTtl: 0,
-        includeStatus: true
-      }
-    })
+    spark.internal.presence.unsubscribe(subjects)
       .then(() => {
         spark.internal.mercury.off(`event:apheleia.subscription_update`);
```

Comment:
Would this clear all listeners instead of just the subjects you want to unsubscribe from?